Simbody/Visualizer/include/simbody/internal/Visualizer.h

```diff
@@ -760,6 +764,44 @@ class SimTK_SIMBODY_EXPORT Visualizer::FrameController {
     virtual ~FrameController() {}
 };
 
+/** Causes the camera to point at and follow a point fixed on a body (a
+ station). This might be useful if your system translates substantially
+ and would thus leave the field of view of a stationary camera. **/
+// This class is based on the BodyWatcher class that used to be in TimsBox.cpp.
+class SimTK_SIMBODY_EXPORT Visualizer::BodyFollower :
+    public Visualizer::FrameController {
+public:
+    /**
+    @param[in] bodyB
```

Comment:
This name doesn't match the formal parameter name in the constructor (`body`). I have been switching to `mobod` for MobilizedBodies to avoid ambiguity regarding the Body class (which I should have called something like `BodyInfo`). So I would suggest `mobodB` for this parameter name.